_licenses/cern-ohl-s-2.0.txt

```diff
@@ -0,0 +1,299 @@
+---
+title: CERN Open Hardware Licence Version 2 - Strongly Reciprocal
+spdx-id: CERN-OHL-S-2.0
+nickname: CERN OHL v2 Strongly Reciprocal
+
+description: The CERN OHL was drafted with hardware in mind.  On top of licensing copyright like software and documentation licenses, it also licenses patents.  This “Strongly Reciprocal” variant applies to a complete design, and if a CERN-OHL-S-2.0 file is used in a project then the full project must be released under the CERN-OHL-S-2.0 (or a compatible license).
```

Comment:
IMO this change makes it clearer that the two clauses are causally related, and not merely concatenated:

```suggestion
description: The CERN OHL was drafted with hardware in mind.  On top of licensing copyright like software and documentation licenses, it also licenses patents.  This “Strongly Reciprocal” variant applies to a complete design, so if a CERN-OHL-S-2.0 file is used in a project then the full project must be released under the CERN-OHL-S-2.0 (or a compatible license).
```